src/Accounts/Accounts/Accounts.csproj

```diff
@@ -24,6 +24,7 @@
     <NetCoreAssemblies Include="$(RepoSrc)lib\Microsoft.IdentityModel.Clients.ActiveDirectory\NetCore\Microsoft.IdentityModel.Clients.ActiveDirectory.Platform.dll" />
     <StorageDependencies Include="$(RepoSrc)lib\WindowsAzure.Storage\9.3.0\Microsoft.WindowsAzure.Storage.dll" />
     <StorageDependencies Include="$(RepoSrc)lib\WindowsAzure.Storage\9.3.0\Microsoft.WindowsAzure.Storage.DataMovement.dll" />
+    <StorageDependencies Include="$(RepoSrc)lib\WindowsAzure.Storage\9.3.0\Mono.Posix.NETStandard.dll" />
```

Comment:
`Microsoft.WindowsAzure.Storage.DataMovement.dll` is this package: https://www.nuget.org/packages/Microsoft.Azure.Storage.DataMovement/0.8.1
The .NetStandard dependencies show `Mono.Posix.NETStandard (>= 1.0.0-beta2)`. It definitely is part of the DependencyMap.csv when you generate a build. It would require further investigation why we haven't needed it previously. If I was to guess, it is because we are running Static Analysis properly now on Linux/macOS and this dependency doesn't apply to Windows.

Reply:
Filed an issue to cover this - there are multiple different platform-specific runtime assemblies for this